test/smoke/smoke_test.go

```diff
@@ -69,37 +69,58 @@ var _ = Describe("Smoke test", func() {
 	Context("Clustered deployment (C3 - clustered indexer, search head cluster)", func() {
 		It("can deploy indexers and search head cluster", func() {
 
-			err := deployment.DeployCluster(deployment.GetName(), 3)
+			idxCount := 3
```

Comment:
Nit: indexerCout, just to avoid confusion with indexes?

Reply:
@sgontla I will resolve this next time I am touching this file.